stdlib/test/utils/test_tuple.mojo

```diff
@@ -98,8 +99,72 @@ def test_tuple_pointer():
     assert_equal(tup[2], 3)
 
 
+def test_array_int():
+    var arr = Array[Int, 3](0, 0, 0)
+
+    assert_equal(arr[0], 0)
```

Comment:
Can we also add tests for negative indexing support, please? Internally, we have a PR for supporting negative indexing, so for parity it'd be good to go ahead and add support here.